metricbeat/module/mongodb/collstats/collstats.go

```diff
@@ -99,22 +85,18 @@ func (m *MetricSet) Fetch(reporter mb.ReporterV2) {
 
 		infoMap, ok := info.(common.MapStr)
 		if !ok {
-			err = errors.New("Unexpected data returned by mongodb")
-			logger.Error(err)
-			reporter.Error(err)
+			m.Logger().Error("Unexpected data returned by mongodb")
```

Comment:
This is a change in behaviour. Previously we also sent an error event now we only log it. I think we should still send the error event.